File: problemas/ad-hoc/3126/3126.swift
import Foundation

let C = Int(readLine(strippingNewline: true)!)!
let numeros = readLine(strippingNewline: true)!.components(separatedBy: " ")

var resposta = 0
for i in 0..<C {
    resposta += Int(numeros[i])!
}

print(resposta)

File: problemas/ad-hoc/3126/3126.cpp
#include <iostream>

using namespace std;

int main(){
    int N, x, resposta;

    cin >> N;
    resposta = 0;
    for(int i = 0; i < N; ++i){
        cin >> x;
        resposta += x;
    }
    cout << resposta << endl;

    return 0;
}